lib/stat_tracker.rb

```diff
@@ -1,102 +1,57 @@
 require 'csv'
-require_relative 'game' 
-require_relative 'teams' 
+require_relative 'game'
+require_relative 'teams'
 
 class StatTracker
-  # attr_reader :highest_total_scored
+
   def initialize(data_hash)
     @games = data_hash[:games]
-  
-  end
+    @teams = data_hash[:teams]
 
+  end
 
   def self.from_csv(database_hash)
+    # first iterate through database and access whichever one you want
     data_hash = {}
     game_array = []
     team_array = []
-    database_hash.map do |key, value| 
-      data_hash[key] = CSV.read(value, headers: true, header_converters: :symbol)
-      data_hash.each do |key, data| 
-        if key == :games
-          data.each do |row|
-           require 'pry'; binding.pry
-          end
+    database_hash.map do |key, value|
+      if key == :games
+        games = CSV.read(database_hash[:games], headers: true, header_converters: :symbol).map do |row|
+          one_game = Game.new(
+                row[:game_id],
+                row[:season],
+                row[:type],
+                row[:away_team_id],
+                row[:home_team_id],
+                row[:away_goals,],
+                row[:home_goals],
+                row[:venue]
+          )
         end
+        game_array << games
+      elsif key == :teams
```

Comment:
nice and now we'll just create our game_teams class and add that here.  so close to building methods!!

Reply:
we'll also make sure to add specs for each class and setup some basic tests in there. so that last PR before async we all have same lib and spec.